src/multiplayer_error_codes.hpp

```diff
@@ -30,6 +30,7 @@
 #define MP_NAME_AUTH_BAN_USER_ERROR    "107"
 #define MP_NAME_AUTH_BAN_IP_ERROR      "108"
 #define MP_NAME_AUTH_BAN_EMAIL_ERROR   "109"
+#define MP_AUTH_SERVER_IP_BAN_ERROR    "110"
```

Comment:
Added a new error code in order to have a separate error message that didn't mention the forums.